src/platform/site-wide/loading-button/LoadingButton.jsx

```diff
@@ -2,16 +2,40 @@ import React from 'react';
 
 export default function LoadingButton({
   isLoading,
+  loadingText,
   children,
   onClick,
   disabled,
   ...props
 }) {
-  const contents = isLoading ? (
-    <i className="fa fa-spinner fa-spin" />
-  ) : (
-    children
-  );
+  let contents;
+  if (loadingText) {
+    contents = isLoading ? (
+      <>
+        <i
+          className="fa fa-spinner fa-spin"
+          aria-hidden="true"
+          role="presentation"
+        />
+        <span className="sr-only" role="status">
+          {loadingText}
+        </span>
+      </>
+    ) : (
+      children
+    );
+  } else {
+    contents = isLoading ? (
+      <i
+        className="fa fa-spinner fa-spin"
+        aria-hidden="true"
+        role="presentation"
+      />
+    ) : (
+      children
+    );
+  }
```

Comment:
Would this be possible instead?  I feel like it is a bit simpler:

```js
  const contents = isLoading ? (
    <>
      <i
        className="fa fa-spinner fa-spin"
        aria-hidden="true"
        role="presentation"
      />
      {loadingText && (
        <span className="sr-only" role="status">
          {loadingText}
        </span>
      )}
    </>
  ) : (
    children
  );
```

Reply:
Modification was suggested by Trevor for a 508 defect.
---
Our VA 508 office contact noticed a situation on the /choose-eye-care page where after pressing the Continue button, our gray spinner button announced itself as "unlabeled zero button unavailable" to JAWS + IE11. I've attempted to re-create it by clearing cache and throttling to slow 3G, but wasn't able to reproduce it. Looking at our code for the platform LoadingButton.jsx I have a proposed code solution. Screen shots attached below.